### ISSUE TITLE
ask zest.releaser not to upload binary wheels

### ISSUE THREAD

**Issue by jugmac00** [zopefoundation/meta#69]
> Do we have to make this change in every repository where we have C code involved?

I'd say yes.  Luckily there aren't that many of them.

_Originally posted by @mgedmin in https://github.com/zopefoundation/persistent/issues/156#issuecomment-789584057_

P.S.: This was a discussion between @mgedmin and @icemac  - I just wanted to help by posting a list of all repos with c-files

**Comment by jugmac00** [comment]
```
❯ all-repos-find-files -C all-repos-zope.json .*\.c$ | grep -v coveragerc
output_zope/zopefoundation/BTrees:BTrees/BTreeItemsTemplate.c
output_zope/zopefoundation/BTrees:BTrees/BTreeModuleTemplate.c
output_zope/zopefoundation/BTrees:BTrees/BTreeTemplate.c
output_zope/zopefoundation/BTrees:BTrees/BucketTemplate.c
output_zope/zopefoundation/BTrees:BTrees/MergeTemplate.c
output_zope/zopefoundation/BTrees:BTrees/SetOpTemplate.c
output_zope/zopefoundation/BTrees:BTrees/SetTemplate.c
output_zope/zopefoundation/BTrees:BTrees/TreeSetTemplate.c
output_zope/zopefoundation/BTrees:BTrees/_IFBTree.c
output_zope/zopefoundation/BTrees:BTrees/_IIBTree.c
output_zope/zopefoundation/BTrees:BTrees/_IOBTree.c
output_zope/zopefoundation/BTrees:BTrees/_IUBTree.c
output_zope/zopefoundation/BTrees:BTrees/_LFBTree.c
output_zope/zopefoundation/BTrees:BTrees/_LLBTree.c
output_zope/zopefoundation/BTrees:BTrees/_LOBTree.c
output_zope/zopefoundation/BTrees:BTrees/_LQBTree.c
output_zope/zopefoundation/BTrees:BTrees/_OIBTree.c
output_zope/zopefoundation/BTrees:BTrees/_OLBTree.c
output_zope/zopefoundation/BTrees:BTrees/_OOBTree.c
output_zope/zopefoundation/BTrees:BTrees/_OQBTree.c
output_zope/zopefoundation/BTrees:BTrees/_OUBTree.c
output_zope/zopefoundation/BTrees:BTrees/_QFBTree.c
output_zope/zopefoundation/BTrees:BTrees/_QLBTree.c
output_zope/zopefoundation/BTrees:BTrees/_QOBTree.c
output_zope/zopefoundation/BTrees:BTrees/_QQBTree.c
output_zope/zopefoundation/BTrees:BTrees/_UFBTree.c
output_zope/zopefoundation/BTrees:BTrees/_UIBTree.c
output_zope/zopefoundation/BTrees:BTrees/_UOBTree.c
output_zope/zopefoundation/BTrees:BTrees/_UUBTree.c
output_zope/zopefoundation/BTrees:BTrees/_fsBTree.c
output_zope/zopefoundation/BTrees:BTrees/sorters.c
output_zope/zopefoundation/persistent:persistent/_timestamp.c
output_zope/zopefoundation/persistent:persistent/cPersistence.c
output_zope/zopefoundation/persistent:persistent/cPickleCache.c
output_zope/zopefoundation/persistent:persistent/ring.c
output_zope/zopefoundation/zope.interface:src/zope/interface/_zope_interface_coptimizations.c
output_zope/zopefoundation/zope.i18nmessageid:src/zope/i18nmessageid/_zope_i18nmessageid_message.c
output_zope/zopefoundation/zope.hookable:src/zope/hookable/_zope_hookable.c
output_zope/zopefoundation/zope.proxy:src/zope/proxy/_zope_proxy_proxy.c
output_zope/zopefoundation/zope.security:src/zope/security/_proxy.c
output_zope/zopefoundation/zope.security:src/zope/security/_zope_security_checker.c
output_zope/zopefoundation/zope.container:src/zope/container/_zope_container_contained.c
output_zope/zopefoundation/zodbpickle:src/zodbpickle/_pickle_27.c
output_zope/zopefoundation/zodbpickle:src/zodbpickle/_pickle_33.c
output_zope/zopefoundation/zope.index:src/zope/index/text/okascore.c
output_zope/zopefoundation/AccessControl:src/AccessControl/cAccessControl.c
output_zope/zopefoundation/Acquisition:src/Acquisition/_Acquisition.c
output_zope/zopefoundation/ExtensionClass:src/ComputedAttribute/_ComputedAttribute.c
output_zope/zopefoundation/ExtensionClass:src/ExtensionClass/_ExtensionClass.c
output_zope/zopefoundation/ExtensionClass:src/ExtensionClass/pickle/pickle.c
output_zope/zopefoundation/ExtensionClass:src/MethodObject/_MethodObject.c
output_zope/zopefoundation/Persistence:src/Persistence/_Persistence.c
```

**Comment by icemac** [comment]
Fixed by #108.
